--- tests/cppblog-blog-summary.expected.cpp	2014-09-04 18:23:20.554350867 +0200
+++ tmp/cppblog-blog-summary.cpp	2014-09-04 18:24:48.595156951 +0200
@@ -239,7 +239,7 @@
 #line 41 "tests/cppblog-blog-summary.tmpl"
 		    }
 #line 43 "tests/cppblog-blog-summary.tmpl"
-		    out() << "\n                </p>\n";
+		    out() << "\n                </p>\n        ";
 #line 43 "tests/cppblog-blog-summary.tmpl"
 		}		// endif
 #line 44 "tests/cppblog-blog-summary.tmpl"
@@ -284,7 +284,7 @@
 #line 50 "tests/cppblog-blog-summary.tmpl"
 		    }
 #line 52 "tests/cppblog-blog-summary.tmpl"
-		    out() << "\n                </p>\n";
+		    out() << "\n                </p>\n        ";
 #line 52 "tests/cppblog-blog-summary.tmpl"
 		}		// endif
 #line 56 "tests/cppblog-blog-summary.tmpl"
