--- tests/examples-uploads-view.expected.cpp	2014-09-01 01:05:47.125023648 +0200
+++ tmp/examples-uploads-view.cpp	2014-09-01 01:09:11.671367816 +0200
@@ -20,7 +20,7 @@
 	virtual void render() {
 #line 10 "tests/examples-uploads-view.tmpl"
 	    out() <<
-		"\n<html>\n   <body>\n    <!-- Specify enctype=\"multipart/form-data\" to be able to upload files -->\n\n    <form method=\"post\" action=\"\" enctype=\"multipart/form-data\" >\n";
+		"\n<html>\n   <body>\n    <!-- Specify enctype=\"multipart/form-data\" to be able to upload files -->\n\n    <form method=\"post\" action=\"\" enctype=\"multipart/form-data\" >\n    ";
 #line 10 "tests/examples-uploads-view.tmpl"
 	    {
 		cppcms::form_context _form_context(out(),
