--- tests/examples-forms-view.expected.cpp	2014-09-04 18:23:20.743011983 +0200
+++ tmp/examples-forms-view.cpp	2014-09-04 18:25:38.916317143 +0200
@@ -43,11 +43,11 @@
 #line 11 "tests/examples-forms-view.tmpl"
 	    } else {
 #line 13 "tests/examples-forms-view.tmpl"
-		out() << "\n    \t<h1>Input your details</h1>\n";
+		out() << "\n    \t<h1>Input your details</h1>\n    ";
 #line 13 "tests/examples-forms-view.tmpl"
 	    }
 #line 15 "tests/examples-forms-view.tmpl"
-	    out() << "\n    <form method=\"post\" action=\"\" >\n";
+	    out() << "\n    <form method=\"post\" action=\"\" >\n    ";
 #line 15 "tests/examples-forms-view.tmpl"
 	    {
 		cppcms::form_context _form_context(out(),
