--- tests/examples-sessions-view.expected.cpp	2014-09-04 18:23:20.853064301 +0200
+++ tmp/examples-sessions-view.cpp	2014-09-04 18:26:52.832706367 +0200
@@ -35,15 +35,15 @@
 #line 9 "tests/examples-sessions-view.tmpl"
 		out() << cppcms::filters::escape(content.age);
 #line 11 "tests/examples-sessions-view.tmpl"
-		out() << "</p>\n        <h2>Change details</h2>\n";
+		out() << "</p>\n        <h2>Change details</h2>\n    ";
 #line 11 "tests/examples-sessions-view.tmpl"
 	    } else {
 #line 13 "tests/examples-sessions-view.tmpl"
-		out() << "\n        <h2>Input your details</h2>\n";
+		out() << "\n        <h2>Input your details</h2>\n    ";
 #line 13 "tests/examples-sessions-view.tmpl"
 	    }
 #line 15 "tests/examples-sessions-view.tmpl"
-	    out() << "\n    <form method=\"post\" action=\"\" >\n";
+	    out() << "\n    <form method=\"post\" action=\"\" >\n    ";
 #line 15 "tests/examples-sessions-view.tmpl"
 	    {
 		cppcms::form_context _form_context(out(),
