--- tests/examples-localization-view.expected.cpp	2014-09-04 18:23:20.797514083 +0200
+++ tmp/examples-localization-view.cpp	2014-09-04 18:25:59.747650626 +0200
@@ -25,11 +25,11 @@
 	    if ((cppcms::locale::translate("LTR").str(out().getloc()) ==
 		 "RTL")) {
 #line 11 "tests/examples-localization-view.tmpl"
-		out() << "\n    <body dir=\"rtl\">\n";
+		out() << "\n    <body dir=\"rtl\">\n    ";
 #line 11 "tests/examples-localization-view.tmpl"
 	    } else {
 #line 13 "tests/examples-localization-view.tmpl"
-		out() << "\n    <body>\n";
+		out() << "\n    <body>\n    ";
 #line 13 "tests/examples-localization-view.tmpl"
 	    }
 #line 14 "tests/examples-localization-view.tmpl"
